src/Auth/OAuth/Twitch.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Auth\OAuth;
+
+use Auth\OAuth;
+
+// Reference Material
+// https://dev.twitch.tv/docs/authentication/getting-tokens-oauth/
+
+class Twitch extends OAuth
```

Comment:
Missing some implementation of some mission critical abstract methods.
Please check what @christyjacob4 wrote about this issue here:
https://github.com/appwrite/appwrite/issues/5#issuecomment-537380277